### DISCUSSION TITLE
EDs edstays table has less data on patients compared to COREs transfers table

### DISCUSSION THREAD

**Discussion by kamleshkshirsagar** [MIT-LCP/mimic-code#1235] [MIMIC-IV-ED]
I have been working on the data from edstays and found out that for a patient subject_id=16809648, there are far more entries in transfers table and even more history than in edstays table. Is edstays table complete or is it starting from a particular date thats the reason missing the data.

**Comment by briangow** [comment]
MIMIC-ED data is not available for 2008-2010 but other MIMIC-IV modules do have data in this time-window. This is why there are ED transfers in `core.transfers` that don’t appear in `edstays` in this case.
